doc/ref_arch/kubernetes/chapters/chapter01.md

```diff
@@ -30,42 +100,18 @@ Kubernetes itself is a “system for automating deployment, scaling, and managem
 - monitoring and logging options
 - etc.
 
-Therefore, when considering software validation, and in NFV scenarios the compliance, verification and certification complexities, more thought is required around how flexible the "Platform" should be, with regards to these add-ons.  i.e. does a single blueprint for a "Kubernetes Platform for NFV" include specific versions of specific projects for each add-on, meaning that if you change from, e.g. Envoy to Fluentd, a new blueprint is created to certify against.
-
-There are a number of different approaches that could be adopted:
-1. Software vendor brings Kubernetes
-    - This option places the operator purely as the infrastructure provider, and the software vendor uses the VIM, or IaaS APIs, to manage the lifecycle of the infrastructure required to create, update and delete Kubernetes clusters ***and*** the software being deployed into Kubernetes
-2. Operator brings Kubernetes
-    - This option places the operator as not only the infrastructure provider, but also a "platform-as-a-service" provider (for Kubernetes Platform). The operator may already be providing platform-as-a-service offerings such as database-as-a-service, or other underlying "enabler services" for the software such as DNS, NTP, mail relay, etc.
```

Comment:
Maybe replace PaaS provider with CaaS provider?

Reply:
I have removed PaaS and simplified the CaaS and CaaS Manager definitions.  Resolving this - any new comments can be on the latest commit.